app/assets/javascripts/client/inventories/AnalysisModalCtrl.js

```diff
@@ -57,24 +83,37 @@
       vm.alerts.splice(index, 1);
     };
 
-    vm.hasInventories = function () {
+    vm.hasInventories = function() {
       return vm.hasFetchedInventories && _.any(vm.inventories);
     };
 
-    vm.hasNoInventories = function () {
+    vm.hasNoInventories = function() {
       return vm.hasFetchedInventories && _.isEmpty(vm.inventories);
-    }
-    vm.save = function () {
+    };
+
+    vm.save = function() {
       Analysis.create(null, vm.analysis)
           .$promise
-          .then(function (productEntry){
+          .then(function(productEntry) {
             vm.closeModal();
```

Comment:
I can double check but it's likely out of scope.  I was using the existing modal code there and it seems that IntelliJ found a bracket that was too close to a paren.